### PR TITLE
PEP8

### PR DESCRIPTION
**Pull request by sciunto**
I just fixed few stuffs. I won't do more in this PR.

### CODE REVIEW

File: skimage/color/colorconv.py [hunk 1]
```diff
@@ -844,10 +844,10 @@ def gray2rgb(image, alpha=None):
             is_rgb = True
 
     if is_rgb:
-        if alpha == False:
+        if alpha is False:
             image = image[..., :3]
 
-        elif alpha == True and not is_alpha:
+        elif alpha is True and is_alpha is False:
```
Comment (soupault) [inline skimage/color/colorconv.py]: I'd prefer `elif alpha is True and not is_alpha:`.

Comment (sciunto) [inline skimage/color/colorconv.py]: but flake8 doesn't...

Comment (soupault) [inline skimage/color/colorconv.py]: Just tried with mine `flake8`: it is OK with both versions.
I do not insist on changing your fix. :)

Comment (sciunto) [inline skimage/color/colorconv.py]: Travis' flake8 says something different. Maybe more recent flaske8 versions are ok with the use of `not`

**Comment by emmanuelle** [comment]
:+1:

**Pull request merged**